docs/manual/scala/guide/broker/MessageBrokerApi.md

```diff
@@ -0,0 +1,82 @@
+# Message Broker API
+
+The Lagom Message Broker API provides a distributed publish-subscribe model that services can use to share data via topics. A topic is simply a channel that allows services to push and pull data. In Lagom, topics are strongly typed, hence both the subscriber and producer can know in advance what the expected data flowing through will be.
+
+## Declaring a topic
+
+To publish data to a topic a service needs to declare the topic in its [[service descriptor|ServiceDescriptors#service-descriptors]].
+
+@[hello-service](code/docs/mb/HelloService.scala)
+
+The syntax for declaring a topic is similar to the one used already to define services' endpoints. The [`Descriptor.withTopics`](api/index.html#com.lightbend.lagom.scaladsl.api.Descriptor) method accepts a sequence of topic calls, each topic call can be defined via the [`topic`](api/index.html#com.lightbend.lagom.scaladsl.api.Service$) method on the `Service` object. The latter takes a topic name (i.e., the topic identifier), and a method reference that returns a [`Topic`](api/index.html#com.lightbend.lagom.scaladsl.api.broker.Topic) instance.
```

Comment:
For scaladocs, it's better to link directly to the scaladoc page, rather than the index, since scaladocs will automatically redirect you to the index with the right page in frames, and as of Scala 2.12, the docs are served without frames anyway. Linking directly to the scaladoc page improves SEO.